### PR TITLE
View web account login

### LINKED ISSUE #6: View Username based on Web Account
Using the terminal, a user should be able to view a username based on an account that they request so that they may use it to log in to that account.

### PR DESCRIPTION
**Pull request by ryanloutos-wustl**

### CODE REVIEW

File: src/websiteAccount/WebsiteAccount.java [hunk 1]
```diff
@@ -81,20 +101,53 @@ public boolean addWebsiteAccount(ConnectionSource databaseConnection) {
...
+			System.out.println("Type the account that you want to see the username for:");
+			while (login.equals("")) {
+				String nickname = keyboard.nextLine();
+				for (WebsiteAccountEntity account : websiteAccounts) {
+					if (nickname.equals(Encryption.decrypt(account.getNickname()))) {
+						login = Encryption.decrypt(account.getWebsiteLogin());
+						System.out.println("Login: " + Encryption.decrypt(account.getWebsiteLogin()));
+					}
+				}
+				if (login.equals("")) {
```
Comment (taliaweiss1) [inline src/websiteAccount/WebsiteAccount.java]: I'd insert here a tiny comment that login would only still be null if the typed in nickname was incorrect. Just a takes a second to understand what's happening because the if statement is in a while loop that ensures that at the beginning of the loop at least login would be null

**Review by taliaweiss1 — COMMENTED** [review]
Code looks great to me - just one comment suggestion to insert. Also I'd add a little more testing in website account test. For testInsertWebsiteAccountToDatabaseWithSameNickname, I'd assertTrue that the first time the account is added that it's successful. I'd also test adding a nickname that's the empty string. For testViewWebsiteAccountLogin I'd do same thing, test the empty string for login in addition to an account that doesn't exist. Great job!

**Review by taliaweiss1 — APPROVED** [review]
Great updates!
